elide-core/src/main/java/com/yahoo/elide/Elide.java

```diff
@@ -244,6 +245,14 @@ protected ElideResponse handleRequest(boolean isReadOnly, Object opaqueUser,
             return buildErrorResponse(new InvalidURLException(e), isVerbose);
 
         } catch (Exception e) {
+            if (e instanceof ValidationException) {
+                return buildErrorResponse(
+                    new HttpStatusException(HttpStatus.SC_BAD_REQUEST, e.getMessage()) {
```

Comment:
I would check first if there exists `statusCode` on the `ValidationException` and return that as an HTTP response?